_posts/2023-06-20-zkMove-0.2.0:-Things-That-We-Delievered.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "zkMove 0.2.0 -  Things that we delievered"
+#cover-img: /assets/img/blog/blog4.jpg
+#thumbnail-img: /assets/img/blog/blog4.jpg
+date:   2023-06-20 12:00:00 +0800
+categories: release
+---
+
+zkMove [0.2.0](https://github.com/young-rocks/zkmove/releases/tag/v0.2.0) is out!
+
+It's a big step forward compared to v0.1.0. some important features are implemented, circuit performance optimized, and also bugs fixed!
+We're thrilled to introduce to you what we have achieved in the last three months.
```

Comment:
We are delighted to announce the release of zkMove [0.2.0](https://github.com/young-rocks/zkmove/releases/tag/v0.2.0), which represents a significant accomplishment in our development journey. This release marks the achievement of our previous goal: full bytecode compatibility with Move.